### ISSUE TITLE
requestMtu seems like it can sometimes hang on some phones

### ISSUE THREAD

**Issue by geeksville** [meshtastic/Meshtastic-Android#27]
... and never call the GATT completion handler (including pixel3).  need to add a separate timeout.  Possibly apply same logic to characteristic reads and writes

**Comment by geeksville** [comment]
```
2020-05-24 08:51:06.081 32319-32382/com.geeksville.mesh D/BluetoothGatt: onClientConnectionState() - status=0 clientIf=6 device=24:6F:28:B2:00:B6
2020-05-24 08:51:06.081 32319-32382/com.geeksville.mesh I/com.geeksville.mesh.service.SafeBluetooth: new bluetooth connection state 2, status 0
2020-05-24 08:51:06.081 32319-32382/com.geeksville.mesh D/com.geeksville.mesh.service.SafeBluetooth: work connect is completed, resuming status=0, res=kotlin.Unit
2020-05-24 08:51:06.081 32319-32382/com.geeksville.mesh I/com.geeksville.mesh.service.RadioInterfaceService: Connected to radio!
2020-05-24 08:51:06.081 32319-32382/com.geeksville.mesh D/BluetoothGatt: refresh() - device: 24:6F:28:B2:00:B6
2020-05-24 08:51:06.082 32319-32382/com.geeksville.mesh D/com.geeksville.mesh.service.SafeBluetooth: Enqueuing work: reqMtu
2020-05-24 08:51:06.082 32319-32382/com.geeksville.mesh D/com.geeksville.mesh.service.SafeBluetooth$BluetoothContinuation: Starting work: reqMtu
2020-05-24 08:51:06.082 32319-32382/com.geeksville.mesh D/BluetoothGatt: configureMTU() - device: 24:6F:28:B2:00:B6 mtu: 512
2020-05-24 08:51:07.166 32319-32319/com.geeksville.mesh D/com.geeksville.mesh.ui.BTScanModel: onScanResult BTScanEntry(name=...f38, addr=...:3A)
```

**Comment by geeksville** [comment]
fixed in 28023b8f427307e88da017a16f7fe9d86c0914d3
